ql/src/test/queries/clientpositive/vector_unix_timestamp.q

```diff
@@ -1,6 +1,6 @@
 
 create table t (d string);
-insert into t values('2020-11-16 22:18:40 UTC');
+insert into t values('2020-11-16 22:18:40');
```

Comment:
Why is it changed?

Reply:
Since with introduction of DateTimefomatter we only parse value which fall under the pattern "yyyy-MM-dd HH:mm:ss" other then that we just drop. But after we have introduce the resolverstyle.strict it will strictly check for the pattern "yyyy-MM-dd HH:mm:ss" . Other then that it will fail and return run. since in function like unix_timestamp() we can provide the pattern to resolve the date/Time. So i have added both the test cases for better understanding.